todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
-
-from todo_app.flask_config import Config
+import os.path
+from flask import Flask,escape,request,Response,render_template, redirect
+import todo_app.data.session_items as session
 
 app = Flask(__name__)
-app.config.from_object(Config)
+app.secret_key = "super secret key"
```

Comment:
You have made this **less secure** by hardcoding it in the codebase. It should have been looking for it in an environment variable. It automatically gets it from the untracked .env file, which you should have created following the README instructions. (It also requires the python-dotenv package but that should have been installed when you ran `poetry install`)